libraries/core-react/src/components/Card/CardMedia.tsx

```diff
@@ -11,21 +11,18 @@ export type CardMediaProps = {
 } & HTMLAttributes<HTMLDivElement>
 
 const StyledCardMedia = styled.div<CardMediaProps>`
-  display: flex;
-  width: 100%;
+  display: grid;
+  place-items: center;
   &:last-child {
-    margin-bottom: 8px;
+    padding-bottom: 24px;
```

Comment:
Should images still have a bottom spacing of `24px` when they are last in the Card? @equinor/eds-ux

Reply:
This will be looked at in #1713